lib/src/text-input/TextInput.test.js

```diff
@@ -394,7 +389,6 @@ describe("TextInput component tests", () => {
     expect(options[0].getAttribute("aria-selected")).toBeNull();
   });
 });
-
```

Comment:
Don't remove this space, have a line jump between describes to make the code more readable.